src/main/java/org/broadinstitute/hellbender/tools/gvs/extract/ExtractCohortLite.java

```diff
@@ -141,14 +141,14 @@ public enum SensitivityFilteringType { GENOTYPE, SITES, NONE }
 
     @Argument(
             fullName ="snps-truth-sensitivity-filter-level",
-            doc = "The truth sensitivity level at which to start filtering SNPs",
+            doc = "The truth sensitivity level above which to start filtering SNPs",
```

Comment:
isn't it...below?

Reply:
Actually no. For VQS_LOD it was below, but for calibration_sensitivity it's flipped - so we filter gts/sites which have calibration_sensitivity > the truth-sensitivity-filter-level